src/main/bash/sdkman-availability.sh

```diff
@@ -85,10 +85,10 @@ function __sdkman_update_broadcast {
 	if [[ "$SDKMAN_AVAILABLE" == "true" && "$broadcast_live_id" != "$broadcast_old_id" && "$COMMAND" != "selfupdate" && "$COMMAND" != "flush" ]]; then
 		mkdir -p "${SDKMAN_DIR}/var"
 
-		echo "$broadcast_live_id" > "$broadcast_id_file"
+		echo "$broadcast_live_id" >| "$broadcast_id_file"
```

Comment:
I do prefer the slightly more verbose syntax using `tee` here. Mind changing that?

Reply:
Yes, I will do